.circleci/config.yml

```diff
@@ -152,10 +144,17 @@ jobs:
       name: go/default
       tag: "1.13"
     steps:
+      - run:
+          name: Export environment variables persistent in execution shell
```

Comment:
As the name of the step says, here we are persisting environment variables for all the steps in this job.